i18n/bundle/bundle.go

```diff
@@ -134,6 +134,25 @@ func (b *Bundle) Translations() map[string]map[string]translation.Translation {
 	return b.translations
 }
 
+// LanguageTags returns the tags of all languages that that have been added.
+func (b *Bundle) LanguageTags() (tags []string) {
```

Comment:
Please don't use named return variables for functions that only have one return variable. I would rather see the variable declaration in the function implementation.

```go
var tags []string
```